thunder/tests/opinfos.py

```diff
@@ -2162,6 +2164,20 @@ def elementwise_binary_generator(op, device, dtype, requires_grad, *, no_rhs_num
         d = number(**kwargs)
         yield SampleInput(c, d)
 
+    if not no_weak_dtypes:
+        tdtype = datatypes._torch_to_thunder_dtype_map[dtype]
+        weak_dtype_table = {
+            datatypes.signedinteger: torch.int64,
+            datatypes.unsignedinteger: torch.uint8,
+            datatypes.floating: torch.float32,
+            datatypes.complexfloating: torch.complex64,
+            datatypes.bool_: torch.bool,
+        }
+
+        e = make((4, 4), **kwargs)
+        f = make((), **kwargs, dtype=weak_dtype_table[type(tdtype)])
```

Comment:
I think this can use this function:

```
https://github.com/Lightning-AI/lightning-thunder/blob/150d7e0bebda9c2f27bddcf7e21206d0c7d37ee9/thunder/core/dtypes.py#L598
```